src/bundle/Resources/public/js/CKEditor/anchor/anchor-editing.js

```diff
@@ -20,14 +20,24 @@ class IbexaAnchorEditing extends Plugin {
     }
 
     init() {
-        const { model } = this.editor;
+        const { commands, model } = this.editor;
 
         model.schema.extend('$block', { allowAttributes: 'anchor' });
         model.schema.extend('embed', { allowAttributes: 'anchor' });
         model.schema.extend('embedInline', { allowAttributes: 'anchor' });
         model.schema.extend('embedImage', { allowAttributes: 'anchor' });
         model.schema.extend('customTag', { allowAttributes: 'anchor' });
 
+        commands.get('enter').on('afterExecute', () => {
```

Comment:
hmm on every enter is dangerous.
What about this flow:
1. Add custom tag with content
2. Add anchor to this custom tag
3. Type something inside custom tag
4. Hit enter
Will this remove anchor from the custom tag, which is not expected?

Reply:
Resolved in private, it works correctly - removes anchor parameter only from active new <p> element